TweetMate/src/main/java/com/ybsystem/tweetmate/models/entities/twitter/TwitterUserList.java

```diff
@@ -0,0 +1,57 @@
+package com.ybsystem.tweetmate.models.entities.twitter;
+
+import com.ybsystem.tweetmate.models.entities.Entity;
+
+import java.util.Date;
+
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import twitter4j.UserList;
+
+@Data
+@EqualsAndHashCode(callSuper = false)
+public class TwitterUserList extends Entity {
+
+    private TwitterUser user;
+    private Date createdAt;
+    private long id;
+    private String name;
+    private String fullName;
+    private String description;
+
+    // Count
+    private int memberCount;
+    private int subscriberCount;
+
+    // State
+    private boolean isProtected;
+    private boolean isFollowing;
+
+    // Not use
+    // private String uri;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://192.168.11.19:9000/coding_rules#rule_key=squid%3ACommentedOutCodeLine)